core/src/i18n/en.yaml

```
@@ -1102,13 +1112,18 @@ namespaces:
       error: 'Error while loading {{title}} data due to: {{message}}'
       pods: $t(pods.title)
       title: Healthy Resources
+  statuses:
+    active: Active
+    terminating: Terminating
   title: Namespaces
   tooltips:
+    active: The Namespace is available for use in the system.
```

```suggestion
    active: The Namespace is available for the use.
```
Do we need the info that it is available for use `in the system`? Maybe we can omit that?